build_docker.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-
 cd "$( dirname "${BASH_SOURCE[0]}" )"
-docker build -t eu.gcr.io/unit8-product/u8timeseries:latest .
+version=$(head -n 1 u8timeseries/VERSION)
+docker build -t eu.gcr.io/unit8-product/u8timeseries:${version} .
```

Comment:
Would also leave the latest tag, since we can have multiple:

docker build -t eu.gcr.io/unit8-product/u8timeseries:${version} -t eu.gcr.io/unit8-product/u8timeseries:latest .

Reply:
done